src/custom-elements/src/components/brand/brand.tsx

```diff
@@ -0,0 +1,58 @@
+    }
+  }
+
+  render() {
+    return (
+      <a class={`a-brand ${this.size ? `-${this.size}` : ''}`} href="#" aria-label="CenturyLink">
```

Comment:
@srinath-koneru I think it doesn't make sense to have an anchor element inside the component that cannot have its `href` attribute configured. In my opinion it will be better to change the `a` element with a `div`, a `span` or remove it if is possible. The user will be able to wrap the brand element with an anchor.

Reply:
user can perform @click on chi-spinner he doesn't need to wrap it around  using ```a```  tag to change the location or navigate to a diff route. Changing it to ```span``` though.